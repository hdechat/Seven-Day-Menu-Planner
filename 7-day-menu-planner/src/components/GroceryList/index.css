.grocery-list {
  margin: 1rem auto;
  width: 9rem;
  width: 95%;
  margin: auto;
  height: 500px;
}

.grocery-list h2 {
  margin-bottom: 0;
}

.groceries {
  background: ghostwhite;
  border-style: inset;
  height: 500px;
  list-style: none;
  margin: auto;
  overflow-y: scroll;
  padding: 2rem;
  text-align: left;
  width: 50%;
}

.groceries li {
  height: 1rem;
  margin: 0;
}

.food-item p {
  color: black;
  width: 50rem;
}

.print {
  font-size: 15px;
  height: 2rem;
  left: 20.7rem;
  margin-bottom: 0.2rem;
  padding-top: 5px;
  position: relative;
}

@media print {
    h2 {
      margin-bottom: 0;
    }

    header, .print {
      display: none;
    }
    .groceries {
        border-style: none;
        display: block !important;
        overflow: visible;
    }
}