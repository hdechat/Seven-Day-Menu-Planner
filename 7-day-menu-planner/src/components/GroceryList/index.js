import React from 'react';

const GroceryList = ({ groceryList }) => {
  const groceries = groceryList.map((foodItem, index) => {
    return (
      <p key={foodItem[0] + foodItem[1] + index}>{foodItem}</p>
    )
  });
  return (
    <section className="grocery-list">
      <h1>GROCERY LIST</h1>
      { groceries }
    </section>
  )
}

export default GroceryList;