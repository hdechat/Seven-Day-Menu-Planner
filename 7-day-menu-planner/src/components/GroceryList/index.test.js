import React from 'react';
import { shallow } from 'enzyme';
import GroceryList from './index.js';

describe('GroceryList', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<GroceryList groceryList={[]} />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});