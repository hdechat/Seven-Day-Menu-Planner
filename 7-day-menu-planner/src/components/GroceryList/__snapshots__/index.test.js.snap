// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroceryList matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GroceryList
    groceryList={Array []}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          GROCERY LIST
</h1>,
        Array [],
      ],
      "className": "grocery-list",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "GROCERY LIST",
        },
        "ref": null,
        "rendered": "GROCERY LIST",
        "type": "h1",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            GROCERY LIST
</h1>,
          Array [],
        ],
        "className": "grocery-list",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "GROCERY LIST",
          },
          "ref": null,
          "rendered": "GROCERY LIST",
          "type": "h1",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
